dlrs/clearlinux/pytorch/mkl/licenses/clear_LICENSE

```diff
@@ -43,6 +43,7 @@ BSD-3-Clause-LBNL
 BSD-4-Clause
 BSD-4-Clause-UC
 BSL-1.0
+Beerware
```

Comment:
Thats first time i have heard about this License, from now on gonna use that instead for my projects :D